src/modules/roc_core/target_posix_time/roc_core/time.cpp

```diff
@@ -49,6 +63,26 @@ void sleep_for_ms(uint64_t ms) {
         }
     }
 }
+#else
+void sleep_until_ms(uint64_t ms) {
+    uint64_t now = timestamp_ms();
+    if (ms > now) {
+        sleep_for_ms(ms - now);
+    }
+}
+
+void sleep_for_ms(uint64_t ms) {
```

Comment:
I think we need separate #ifdefs for TIMER_ABSTIME and CLOCK_MONOTONIC, because they are actually separate features.